pegasus/sites.v3/code.org/public/teacher-dashboard/section_progress.haml

```diff
@@ -73,6 +73,9 @@ content-type: text/ng-template
       %dt
         %span.level_link.perfect &nbsp;
       %dd= I18n.t('dashboard_progress_completed_perfect')
+      %dt
+        %span.level_link.submitted &nbsp;
+      %dd= I18n.t('dashboard_progress_submitted')
```

Comment:
This is pegasus, so should be added to i18n gsheet. To test locally add to pegasus/cache/i18n/en-US.yml.